pkg/sdkserver/localsdk.go

```diff
@@ -585,6 +589,59 @@ func (l *LocalSDKServer) GetPlayerCapacity(_ context.Context, _ *alpha.Empty) (*
 	return result, nil
 }
 
+// GetCounter returns a Counter. Returns NOT_FOUND if the counter does not exist.
+// [Stage:Alpha]
+// [FeatureFlag:CountsAndLists]
+func (l *LocalSDKServer) GetCounter(ctx context.Context, in *alpha.GetCounterRequest) (*alpha.Counter, error) {
+	if !runtime.FeatureEnabled(runtime.FeatureCountsAndLists) {
+		return nil, errors.Errorf("%s not enabled", runtime.FeatureCountsAndLists)
+	}
+
+	l.logger.Info("Getting Counter")
+	l.recordRequest("getcounter")
+	l.gsMutex.RLock()
+	defer l.gsMutex.RUnlock()
+
+	if counter, ok := l.gs.Status.Counters[in.Name]; ok {
+		return &alpha.Counter{Name: in.Name, Count: &counter.Count, Capacity: &counter.Capacity}, nil
+	}
+	return nil, status.Errorf(codes.NotFound, "%s Counter NOT_FOUND", in.Name)
```

Comment:
It's weird to see `status` introduced where it's just not present in the API - even line 597 isn't using a status return. I'm not saying it's wrong but do we need a plan to convert over to full status, or should be keep it consistent with what's already here?

Regardless, the second half after `codes.NotFound` is for humans - just use prose instead of `NOT_FOUND`

Reply:
I introduced it as part of the https://google.aip.dev/193 `Services must return a [google.rpc.Status](https://github.com/googleapis/googleapis/blob/master/google/rpc/status.proto) message when an API error occurs, and must use the canonical error codes defined in [google.rpc.Code](https://github.com/googleapis/googleapis/blob/master/google/rpc/code.proto).` 

Do you have a strong preference for keeping the errors as-is?

Reply:
The best thing about the AIPs is at any time you can just pull up https://google.aip.dev/200 to ignore them. :)

I personally think you should just file an issue on this one and stay consistent with existing patterns. It does look like it would be straightforward to move the whole SDK, since we're not using a special error type or anything. But I think it should be done as a follow-on, possibly enforced with a helper in testing to prevent "error type leaks".

But as a more concrete point, if you do introduce them for this particular new API, I would fix line 597 at least.

Reply:
How much consistency are we looking for? 

If we need to change all errors that are passed through (creating gameserver line 116, decoding line 715, etc.) then it seems excessive to change everything for the sake of adhering to guidelines. In this case I would just leave the error formatting as-is.

But if we're only looking to consistency in errors that are created within the SDK (i.e. lines 412, 430) then most of these are related to the PlayerTracking which will be removed at some point in the future. In this case I would use the status.Error instead of the errors.Error since we'll be removing the PlayerTracking code.

Reply:
For me, consistency is more important than adhering to guidelines. I would rather not piecemeal adhere to guidelines, but I do think there's an opportunity to fix line 116 / 715 etc. if you want _later_. So I'm proposing either (a) ignoring aip-193 in your code, or (b) adhering to it and filing an issue for follow-on to fix the rest.

Reply:
Sounds good -- I'll just remove the references to status.Error and keep consistent with error.Error.